fiftyone/server/query.py

```diff
@@ -204,10 +204,17 @@ class CustomizeColor:
     value_colors: t.Optional[t.List[LabelSetting]]
 
 
+@gql.type
+class LabelTagColor:
+    fieldColor: t.Optional[str]
+    valueColors: t.Optional[t.List[LabelSetting]]
```

Comment:
Please only use snake case in Python
```
```
```suggestion
    field_color: t.Optional[str]
    value_colors: t.Optional[t.List[LabelSetting]]
```